### DISCUSSION TITLE
Próximos sistemas a serem incorporados

### DISCUSSION THREAD

**Discussion by rfsaldanha** [rfsaldanha/microdatasus#31] [Announcements]
1. SINAN/Dengue. Branch criado, em programação.
2. SINAN/Chikungunya
3. SINAN/Zika
4. SINAN/Malária
5. SINAN/Febre Amarela

**Comment by richila** [comment]
E o CMD? No site sempre dá dados não encontrados.

**Reply by rfsaldanha** [reply]
Olá! Pelo o que sei, a iniciativa do Conjunto Mínimo de Dados foi descontinuado pelo DataSUS.

**Comment by theozindodatasus** [comment]
Como posso fazer a conversão do SINAN- DENGUE?
